utils/handleServerErrorMessaging.ts

```diff
@@ -0,0 +1,18 @@
+import type { ToastInterface } from 'vue-toastification'
+import type { ServerError } from '~/typedefs/serverResponse'
+
+export function handleServerErrorMessaging(
```

Comment:
nit: you could put this in the graphql file since I don't think it'll be used outside of it

Reply:
Due to toast being accessed on build and also translation not be available. We talked about this. This can be addressed in a future issue if we can figure out a better way to load toasts not on mount and have translations available outside of the components.